### ISSUE TITLE
Is it possible to specify projection of bbox?

### ISSUE THREAD

**Issue by joethorley** [poissonconsulting/fwapgr#21]

**Comment by joethorley** [comment]
This may not be a good idea - simply for discussion

**Comment by sebdalgarno** [comment]
yes if we require an object of class "bbox", i.e. as generated by `sf::st_bbox(x)`...this stores projection info. I think this is a reasonable thing to ask of a user as i think the bbox will be generated by sf most of the time. It will be inconvenient if the user simply has 4 numbers for the bbox that they got some other way.

**Comment by sebdalgarno** [comment]
moving to fwatlasbc as I don't think fwapgr should do any bbox conversion
